velox/docs/functions/spark/datetime.rst

```diff
@@ -333,3 +333,8 @@ These functions support TIMESTAMP and DATE input types.
     part of the 53rd week of year 2004, so the result is 2004. Only supports DATE type.
 
         SELECT year_of_week('2005-01-02'); -- 2004
+.. spark:function:: date_trunc(format, timestamp) -> timestamp
+
+    Returns timestamp ts truncated to the unit specified by the format model.
```

Comment:
Please clarify which values are supported for the 'format'.

Reply:
Added